locals.tf

```diff
@@ -52,6 +54,10 @@ locals {
     iam_role_path                 = var.iam_role_path
     iam_role_permissions_boundary = var.iam_role_permissions_boundary
 
+    # Service IPv4/IPv6 CIDR range
+    service_ipv6_cidr = local.service_ipv6_cidr
```

Comment:
why add variables just to immediately convert them to locals? can't we just use them as variables as is?

Reply:
Good point. Will update that.